lib/nexmo_developer/app/services/translator/file_translator.rb

```diff
@@ -0,0 +1,48 @@
+module Translator
+  class FileTranslator
+    attr_reader :doc_path
+
+    def initialize(doc_path)
+      @doc_path = doc_path
+    end
+
+    def translation_requests
+      doc = load_doc
+      requests = []
+
+      I18n.available_locales.each do |locale|
+        requests << Translator::TranslationRequest.new(locale: locale, frequency: frequency_calculator(doc), path: doc_path)
```

Comment:
can we move the frequency calculator to `TranslationRequest`

Reply:
Why? This doesn't actually calculate the frequency, it just stores it in the object for later calculation. I can rename the method to be clearer, if that'll help.